app/helpers/schemes_helper.rb

```diff
@@ -10,7 +10,6 @@ def display_scheme_attributes(scheme, user)
       { name: "Support services provided by", value: scheme.arrangement_type },
       { name: "Primary client group", value: scheme.primary_client_group },
       { name: "Has another client group", value: scheme.has_other_client_group },
-      { name: "Secondary client group", value: scheme.secondary_client_group },
```

Comment:
Would this with `.compact` at the end work? 
`scheme.has_other_client_group == "Yes" ? { name: "Level of support given", value: scheme.support_type }: nil,`

Reply:
🔥 

if that works you have also removed all the ifs that come after the initial array definition so you can remove saving in variable and just return directly

Reply:
tasty, will give it a go

Reply:
have done both of these 👍